{
  "dataType": "CVE_RECORD",
  "containers": {
    "cna": {
      "title": "IBM App Connect Enterprise denial of service",
      "source": {
        "discovery": "UNKNOWN"
      },
      "metrics": [
        {
          "format": "CVSS",
          "cvssV3_1": {
            "scope": "UNCHANGED",
            "version": "3.1",
            "baseScore": 6.5,
            "attackVector": "NETWORK",
            "baseSeverity": "MEDIUM",
            "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
            "integrityImpact": "NONE",
            "userInteraction": "NONE",
            "attackComplexity": "LOW",
            "availabilityImpact": "HIGH",
            "privilegesRequired": "LOW",
            "confidentialityImpact": "NONE"
          },
          "scenarios": [
            {
              "lang": "en",
              "value": "GENERAL"
            }
          ]
        }
      ],
      "affected": [
        {
          "cpes": [
            "cpe:2.3:a:ibm:app_connect_enterprise:11.0.0.1:*:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:app_connect_enterprise:11.0.0.25:*:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:app_connect_enterprise:12.0.1.0:*:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:app_connect_enterprise:12.0.12.0:*:*:*:*:*:*:*"
          ],
          "vendor": "IBM",
          "product": "App Connect Enterprise",
          "versions": [
            {
              "status": "affected",
              "version": "11.0.0.1",
              "versionType": "semver",
              "lessThanOrEqual": "11.0.0.25"
            },
            {
              "status": "affected",
              "version": "12.0.1.0",
              "versionType": "semver",
              "lessThanOrEqual": "12.0.12.0"
            }
          ],
          "defaultStatus": "unaffected"
        }
      ],
      "references": [
        {
          "url": "https://www.ibm.com/support/pages/node/7154607",
          "tags": [
            "vendor-advisory"
          ]
        },
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/289647",
          "tags": [
            "vdb-entry"
          ]
        }
      ],
      "x_generator": {
        "engine": "Vulnogram 0.2.0"
      },
      "descriptions": [
        {
          "lang": "en",
          "value": "IBM App Connect Enterprise 11.0.0.1 through 11.0.0.25 and 12.0.1.0 through 12.0.12.0 integration nodes could allow an authenticated user to cause a denial of service due to an uncaught exception.  IBM X-Force ID:  289647.",
          "supportingMedia": [
            {
              "type": "text/html",
              "value": "IBM App Connect Enterprise 11.0.0.1 through 11.0.0.25 and 12.0.1.0 through 12.0.12.0 integration nodes could allow an authenticated user to cause a denial of service due to an uncaught exception.  IBM X-Force ID:  289647.",
              "base64": false
            }
          ]
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "type": "CWE",
              "cweId": "CWE-248",
              "description": "CWE-248 Uncaught Exception"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "shortName": "ibm",
        "dateUpdated": "2024-05-22T18:34:39.881Z"
      }
    },
    "adp": [
      {
        "metrics": [
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2024-31904",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "no"
                  },
                  {
                    "Technical Impact": "partial"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-05-23T15:48:27.874415Z"
              }
            }
          }
        ],
        "providerMetadata": {
          "shortName": "CISAADP",
          "orgId": "8c464350-323a-4346-a867-fc54517fa145",
          "dateUpdated": "2024-05-23T19:01:25.733Z"
        }
      }
    ]
  },
  "cveMetadata": {
    "cveId": "CVE-2024-31904",
    "state": "PUBLISHED",
    "dateUpdated": "2024-05-22T18:34:39.881Z",
    "dateReserved": "2024-04-07T12:45:07.197Z",
    "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
    "datePublished": "2024-05-22T18:34:39.881Z",
    "assignerShortName": "ibm"
  },
  "dataVersion": "5.1"
}